site/content/docs/v1.22.1/deploy-options.md

```diff
@@ -227,7 +227,7 @@ $ kubectl delete ns contour-operator
 [4]: /guides/proxy-proto.md
 [5]: https://github.com/kubernetes-up-and-running/kuard
 [7]: {{< param github_url>}}/tree/{{< param version >}}/examples/contour/02-service-envoy.yaml
-[8]: /getting-started.md
+[8]: /getting-started
```

Comment:
could you also make the same change in this file as well https://github.com/projectcontour/contour/blob/main/site/content/docs/main/deploy-options.md

(so that we have it reflected in the docs for future versions)

Reply:
Sorry for that. Done!

Reply:
good catch @sunjayBhatia!